chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/SearchRequest.java

```diff
@@ -144,4 +147,62 @@ public Boolean getHasHadSurveillance() {
 	public void setHasHadSurveillance(Boolean hasHadSurveillance) {
 		this.hasHadSurveillance = hasHadSurveillance;
 	}
+	
+	@Override
+	public int hashCode() {

```

Comment:
Serialization is required for an object to be placed into cache. There is no downside to serializing an object. In order to prevent errors when caching now and in the future, I serialized most of the classes.

hashCode needs to be overridden in order to get a unique hashCode for SearchRequest depending on the values of each of its properties. This is what allows someone to make a /search and have their unique search be cached. I believe I found the algorithm online in a tutorial.